test/ui/shared/test-url-util.js

```diff
@@ -0,0 +1,61 @@
+/* global describe, it */
```

Comment:
The test folder has its own eslintrc. Should probably add this to the environment, or create a globals array.

But I'm not even sure we need those rules. Aren't those part of the mocha environment, which we already specify?